commands/basic/join.js

```diff
@@ -1,22 +1,23 @@
-const {prefix} = require("./../../config");
+const {prefix} = require("./../../config")
 
-join = (client) => {
+const join = (client) => {
     client.on("message", (message) => {
         if (!message.guild){
             return
         }
         if(message.author.bot){
             return
         }
-        msg = message.content.toLowerCase()
+        const msg = message.content.toLowerCase()
+        
         if(msg === prefix + "join" || msg === prefix + "j"){
             if(message.member.voiceChannel){
                 message.member.voiceChannel.join()
-                .then((connection) => {
-                    const rng = Math.floor((Math.random() * 3));
-                    message.reply(["joined baby!", "right here!", "up and running!"][rng]);
-                })
-                .catch((e) => (message.reply("Im not able to join that channel. Adjust your perms moron.")));
+                    .then(() => {
+                        const rng = Math.floor((Math.random() * 3))
+                        message.reply(["joined baby!", "right here!", "up and running!"][rng])
+                    })
+                    .catch((e) => (message.reply(`Im not able to join that channel. Adjust your perms moron. ${e}`)))
```

Comment:
lol

Reply:
Its important to let the person know that they're an idiot. You have to keep them down